content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,245 @@
+
+Dedicated, single-tenant Sourcegraph Cloud instances are now generally available, and the best way for teams to use Sourcegraph. You can read more in our announcement blog [here](/blog/enterprise-cloud), or [sign up for a 30-day trial for your organization](https://signup.sourcegraph.com/).
+
+<a href="https://docs.sourcegraph.com/cloud" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+  
+<br />
+<Badge link="https://docs.sourcegraph.com/cloud" text="Admin" color="violet" size="small" />
+
+#### SOC 2 Type II certification for Sourcegraph Cloud
+
+We know that your code is one of your most important and sensitive assets. Every component of Sourcegraph was designed with security in mind. As part of our ongoing commitment to security, we recently received our SOC 2 Type II attestation for Sourcegraph Cloud. When you use Sourcegraph, you can be confident that Sourcegraph controls the security and confidentiality of your sensitive data.
+
+Check out our [Security Portal](https://security.sourcegraph.com/) to access the report and learn more about our security program.
+
+<a href="https://security.sourcegraph.com/" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### 📈 Advanced admin capabilities
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" text="Admin" color="violet" size="small" />
+
+#### Upgrade directly to Sourcegraph 4.0 from earlier versions of Sourcegraph
+We know that upgrading Sourcegraph can be a time-intensive process, especially if you fall a few versions behind. Sourcegraph 4.0 supports [multi-version upgrades](https://docs.sourcegraph.com/admin/updates#multi-version-upgrade) so you can upgrade to 4.0 directly from version 3.20 or higher.
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Multi-version%20upgrades%20final',
+    mp4: 'blog/release-post/4.0/Multi-version%20upgrades%20final'
+  }}
+  loop={true}
+  title="Multi-version upgrades"
+  caption="Upgrade to 4.0 directly from version 3.20 or higher with multi-version upgrades"
+/>
+
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="https://docs.sourcegraph.com/admin/analytics" text="Admin" color="violet" size="small" />
+
+#### Understand usage and quantify the value of using Sourcegraph with in-product analytics
+[In-product analytics](https://docs.sourcegraph.com/admin/analytics) helps admins understand user engagement, measure efficiency in terms of time saved, and calculate the value of utilizing Sourcegraph.
```

Comment:
```suggestion
[In-product analytics](https://docs.sourcegraph.com/admin/analytics) helps admins understand user engagement, measure efficiency in terms of time saved, and calculate the value of utilizing Sourcegraph.

These analytics show the exact usage of your Sourcegraph instance so you can see which features are most used across your organization. The usage data is also used to quantify how much time developers have saved as they use Sourcegraph to search and understand code, run batch changes, create insights, and more. You can also customize the default time-saved values to reflect how your team uses Sourcegraph. 

With in-product analytics, you can now report on:
```

If you incorporate this change, delete The feature value calculators use data derived from event logs and the default time-saved values can be customized to reflect how your team uses Sourcegraph.